spec/rails_helper.rb

```diff
@@ -67,7 +67,6 @@
 
   config.include ApplicationHelper
   config.include ActionMailer::TestHelper
-  config.include ActiveJob::TestHelper
```

Comment:
I did do a check on where we using this, but just want to highlight that I'm removing the ActiveJob Test helper.

Reply:
Thanks @Ridhwana! This should probably have been removed a couple of months ago when we finished moving everything over to Sidekiq.